raiden-ts/src/messages/utils.ts

```diff
@@ -432,10 +435,22 @@ export function validateAddressMetadata(
   metadata: AddressMetadata | undefined,
   address: Address,
   { log }: { log: logging.Logger } = { log: logging },
-): AddressMetadata | undefined {
-  if (metadata && verifyMessage(metadata.user_id, metadata.displayname) === address)
-    return metadata;
-  else if (metadata) log?.warn('Invalid address metadata', { address, metadata });
+): [metadata: AddressMetadata, pubkey: PublicKey] | undefined {
+  if (!metadata) return;
```

Comment:
Before we were logging a warning here, is this intended?

Reply:
I don't think anything was logged if `!metadata`. The log call was only issued if metadata existed, so in the spirit of _the happy case to the left_, I check the undefined case early and just skip in that case.